tests/unit/test_spatial_methods/test_finite_volume.py

```diff
@@ -959,7 +956,12 @@ def test_indefinite_integral(self):
         grad_and_div = pybamm.div(pybamm.grad(phi))
         int_grad_phi = pybamm.IndefiniteIntegral(grad_and_div, x)
         disc.set_variable_slices([phi])
-
+        disc._bcs = {
```

Comment:
why do we now have boundary conditions here?

Reply:
Needed so that the integrand has the right shape, will add a comment